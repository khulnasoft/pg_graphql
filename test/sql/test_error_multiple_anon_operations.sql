select graphql.resolve(
    query:='{ anon1 } { anon2 }'
)
