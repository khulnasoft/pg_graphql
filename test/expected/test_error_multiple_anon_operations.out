select graphql.resolve(
    query:='{ anon1 } { anon2 }'
)
                                       resolve                                        
--------------------------------------------------------------------------------------
 {"errors": [{"message": "Anonymous operations must be the only defined operation"}]}
(1 row)

